src/operator/tensor/elemwise_binary_broadcast_op_basic.cc

```diff
@@ -122,6 +122,8 @@ Example::
 
```

Comment:
Need to clarify what's supported/not supported in doc like https://mxnet.incubator.apache.org/versions/master/api/python/ndarray/sparse.html#mxnet.ndarray.sparse.dot

Reply:
Done

Reply:
Please only add the docs for broadcast_mul here.